docs/PowerAuth-Server-1.9.0.md

```diff
@@ -70,6 +70,12 @@ to `true` (enabled) by default. A Callback URL remains enabled until a delete re
 Management API, at which point the `enabled` column is set to `false` (disabled). Disabled Callback URLs will be
 excluded from all subsequent queries and operations.
 
+### Add Column of Last Callback Configuration Update
+
+A new column `timestamp_last_updated` has been added to the `pa_application_callback` table to store the timestamp of the
```

Comment:
🤔 Shouldn't we create also `timestamp_created` just to have better context for `timestamp_last_updated`?